src/frontend/src/lib/components/core/Menu.svelte

```diff
@@ -39,15 +39,9 @@
 	};
 </script>
 
-<button
-	class="user icon desktop-wide"
```

Comment:
Can we clean-up the style "user"?

Reply:
sure! i would have done in another PR, do you prefer in this one?

Reply:
Another PR is fine, thx

Reply:
Done with PR #1846